### ISSUE TITLE
How to add dreamcast .GDI files?

### ISSUE THREAD

**Issue by gwokk** [scottrice/Ice#443]
Hi, 

Loving the program so far. 

I've just got one problem in that I have a bunch of dreamcast ROMs that are .gdi which are also accompanied by track01.bin, track02.raw & track03.bin files which I normally keep in a named folder. 
However, these aren't detected by Ice in contrast to the .cdi ROMs which are detected fine in the main dreamcast folder. Because the track files are all the same for each ROM, I can't have multiple of the same name in the same folder, is there a way I can get Ice to detect the .gdi ROMs? 

Also mentioned in other issues, consolegrid doesn't seem to allow uploading at the moment. 

I have tried using the 'images directory=C:\' line in consoles.txt and I have placed an image file in the relevant system in both correctly sized jpg and png formats but when running ice again - they don't seem to show up. Is anyone else experiencing this also? 

Cheers

**Comment by Bboy486** [comment]
Did you try extension= cdi, gdi?

**Comment by gwokk** [comment]
Yes, I've got both extensions. But the 5kb gdi files are not in the same folder because the accompanying track files are all the same. I might the gdi only file and leaving the track files in the named game folder. 

Has anyone got local image directory to work properly? Ice won't add my image to steam despite it being directed to the correct folder and in the same name as the ROM
